packages/core/src/Input/Input.tsx

```diff
@@ -410,6 +417,7 @@ function Input<T extends string | NumberInputType>({
 			className={className}
 			width={width}
 			hasError={error !== undefined}
+			onWheelCapture={handleWheelCapture}
```

Comment:
I think only `NumberInput` needs this handler so maybe you should check the input type and only use the handler for `NumberInput`.

Reply:
Good idea :)